odl/space/cu_ntuples.py

```diff
@@ -647,9 +698,9 @@ def _multiply(self, x1, x2, out):
         >>> x2 = rn.element([1, 2, 3])
         >>> out = rn.element()
         >>> rn.multiply(x1, x2, out)  # out is returned
-        CudaRn(3).element([5.0, 6.0, 6.0])
+        CudaRn(3, 'float32').element([5.0, 6.0, 6.0])
```

Comment:
I don't exactly know. It shouldn't be printed out, right? I noted though that this doctest failed and fixed it. Need to look at it.